Sources/PackageDescription/LanguageStandardSettings.swift

```diff
@@ -10,66 +10,126 @@
 
 /// The supported C language standard to use for compiling C sources in the package.
 public enum CLanguageStandard: String, Encodable {
```

Comment:
@abertelrud Should I expand the `CLanguageStandard` documentation comment?

```suggestion
/// The supported C language standard to use for compiling C sources in the package.
///
/// Aliases are available for some C language standards. For example,
/// use `c89`, `c90`, or `iso9899_1990` for the "ISO C 1990" standard.
/// To learn more, read the [Clang Compiler User's Manual][1].
///
/// [1]: <https://clang.llvm.org/docs/UsersManual.html#differences-between-various-standard-modes>
public enum CLanguageStandard: String, Encodable {
```

Reply:
That's great, I think that will be a big help for those who aren't familiar with the naming here and the details of each mode!

Reply:
@abertelrud After the current smoke tests are finished, I'll commit the suggestions.